### DISCUSSION TITLE
How to use avisynth+ on google colaboratory ?

### DISCUSSION THREAD

**Discussion by Hrushikesh-Reddy** [AviSynth/AviSynthPlus#231] [Q&A]
How to use avisynth+ on google colaboratory , I have to remove logo from my videos using delogohd but my cpu does not support sse4.1.

**Comment by pinterf** [comment]
If your CPU does not support sse4.1 and your script (e.g. delogohd) does not work, then one of the external plugins which is used by delogohd is probably written without pre-SSE4.1 support. Avisynth itself is not affected in this problem. You have to find out which 3rd party plugin is affected and ask its developer to change the code and help you.
